src/components/BrowserRow/BrowserRow.react.js

```diff
@@ -20,29 +20,7 @@ export default class BrowserRow extends Component {
   }
 
   render() {
-    const {
-      className,
-      columns,
-      currentCol,
-      isUnique,
-      obj,
-      onPointerClick,
-      onPointerCmdClick,
-      order,
-      readOnlyFields,
-      row,
-      rowWidth,
-      selection,
-      selectRow,
-      setCopyableValue,
-      setCurrent,
-      setEditing,
-      setRelation,
-      onEditSelectedRow,
-      setContextMenu,
-      onFilterChange,
-      markRequiredFieldRow,
-    } = this.props;
+    const { className, columns, currentCol, isUnique, obj, onPointerClick, onPointerCmdClick, order, readOnlyFields, row, rowValue, rowWidth, selection, selectRow, setCopyableValue, setCurrent, setEditing, setRelation, onEditSelectedRow, setContextMenu, onFilterChange, markRequiredFieldRow } = this.props;
```

Comment:
Could you run prettier and eslint? Did looks like a long line that would be refactored